cmd/plan.go

```diff
@@ -243,6 +259,134 @@ func planInstall(rv cran.RVersion, exitOnMissing bool) (*cran.PkgNexus, gpsr.Ins
 	return pkgNexus, installPlan, rollbackPlan
 }
 
+// Tarball manipulation code taken from https://gist.github.com/indraniel/1a91458984179ab4cf80 -- is there a built-in function that does this?
+func unpackTarballs(fs afero.Fs, cfg configlib.PkgrConfig) []desc.Dep {
```

Comment:
rather than passing in the entire config - grab the things you need from the config as specific vars. Eg cache, Tarballs can be separate args. It makes it much more clear around what unPackTarballs needs, and makes it easier to isolate/test etc